specification/migrate/resource-manager/Microsoft.Migrate/AssessmentProjects/readme.nodejs.md

```diff
@@ -0,0 +1,14 @@
+## Node.js
+
```

Comment:
I don't think we are using nodejs.md file now, could you remove all nodejs.md files in this PR?

Reply:
Sure, will remove - thanks.